src/main/java/com/app/persistent/JsonParser.java

```diff
@@ -200,7 +207,7 @@ public static boolean writeNewMeasurement(String autoSavingDir, Measurement meas
             pw.close();
 
             return true;
-        } catch (IOException | ParseException e) {
+        } catch (IOException e) {
             e.printStackTrace();
```

Comment:
@MariaRiya toto nema velmi zmysel, pouzivatel to neuvidí a ty s tym uz nic nerobis, nechces nejaku nitification triggernut?